services/cincinnati-ci/admin_rbac.yaml

```diff
@@ -41,9 +41,8 @@ items:
     metadata:
       name: cincinnati-ci-admins
     users:
-      - steveeJ
       - steveej
-      - lalatendum
+      - lalatendumohanty
```

Comment:
```suggestion
      - LalatenduMohanty
```

Reply:
Fixed in https://github.com/openshift/release/commit/79df66057e329c9ae2f10c7f8c1cef41e281f893